reports/filesDataExporter/batch/batch.go

```diff
@@ -0,0 +1,163 @@
+package batch
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/gen/storage/models"
```

Comment:
could not import github.com/iryonetwork/wwm/gen/storage/models (cannot find package "github.com/iryonetwork/wwm/gen/storage/models" in any of:
	/tmp/tmp.9iNHKsROgt/src/github.com/iryonetwork/wwm/vendor/github.com/iryonetwork/wwm/gen/storage/models (vendor tree)
	/usr/local/go/src/github.com/iryonetwork/wwm/gen/storage/models (from $GOROOT)
	/tmp/tmp.9iNHKsROgt/src/github.com/iryonetwork/wwm/gen/storage/models (from $GOPATH))